packages/core/src/browser/view-container.ts

```diff
@@ -1277,6 +1277,7 @@ export class ViewContainerLayout extends SplitLayout {
         return map(this.items, item => item.widget);
     }
 
+    // @ts-expect-error TS2611 because Phosphor.js implements a getter for `SplitLayout.widgets` but typed it as `readonly widgets`.
```

Comment:
I don't think comment is quite right. I think that older versions of TS, like the one Phosphor was most recently compiled with, didn't distinguish between properties and getters, so the `.d.ts` file shows this like it's a property even though it isn't, and then TS is looking at the `.d.ts` file and seeing a property, even though the class actually has a getter.

Reply:
I tried to be the most concise with what is happening without going into why, but you are totally right.

But it stays true that the typings describe this field as a readonly prop and we try to redefine it as a getter, causing TS2611. I just don't think it is super useful explaining the historical reason in this specific comment.

We could indeed avoid this issue by compiling Phosphor.js using a newer TypeScript, but it isn't worth it.

FYI: https://github.com/eclipse-theia/theia/issues/6501

Reply:
How about `the Phosphor field is declared as a property`. The passive avoids the attribution to the Phosphor authors that 'implements a getter for `SplitLayout.widgets` but typed it as `readonly widgets`' implies.

Reply:
Sounds good to me! Updated.